pgbot/emotion.py

```diff
@@ -102,11 +107,15 @@ async def dad_joke(msg: discord.Message):
                 if char in name:
                     name = name.split(char)[0]
 
-            if name:
+            if name and mode == "happy":
                 await msg.channel.send(
                     f"Hi {name}! I am <@!{common.bot.user.id}>",
                     allowed_mentions=discord.AllowedMentions.none(),
                 )
+            elif name and mode == "depression":
+                await msg.channel.send(
+                    f"Hi {name}! I am... I am.. <:pg_depressed:845321616094789712>"
```

Comment:
We prolly need the `allowed_mentions=discord.AllowedMentions.none(),` block here too